interpreter/Makefile

```diff
@@ -109,12 +109,9 @@ _build/$(LIB).cmxa: $(FILES) $(LIB).mllib _tags Makefile
 # Building JavaScript library
 
 .PHONY:		$(JSLIB)
-$(JSLIB):	$(UNOPT)
-		mkdir -p _build/jslib/src
-		cp meta/jslib/* _build/jslib
-		cp $(DIRS:%=_build/%/*.ml*) meta/jslib/*.ml _build/jslib/src
-		rm _build/jslib/src/*.ml[^i]
-		(cd _build/jslib; ./build.sh ../../$@)
+$(JSLIB):
+		dune build wast.bc.js
+		cp ../_build/default/interpreter/wast.bc.js ./wast.js
```

Comment:
Can you introduce proper build rules for wast.bc.js, so that dependencies and cleanup work?

Reply:
This should be addressed now. I redid the Makefile rule to avoid using dune and just use `ocamlbuild` and calling `js_of_ocaml` directly (there is an ocamlbuild plugin for `js_of_ocaml` but I was unable to get it to work correctly). It should do cleanup now for example.

Happy to fine tune the build rules if anything looks off still.